MirroredTechniquePower crashes the game with a type mismatch error

**Version**
Slay the spire: Main
Mod version: 1.9.1

**Reporting a crash?**
23:02:17.512 INFO basemod.BaseMod> publish on post power apply
23:02:17.513 INFO basemod.BaseMod> powers modified
23:02:23.843 INFO basemod.BaseMod> publish on card use: stsjorbsmod:Withering

Exception in thread "LWJGL Application" com.badlogic.gdx.utils.GdxRuntimeException: java.lang.Error: Unresolved compilation problem: 
	Type mismatch: cannot convert from Object to boolean

	at com.badlogic.gdx.backends.lwjgl.LwjglApplication$1.run(LwjglApplication.java:133)
Caused by: java.lang.Error: Unresolved compilation problem: 
	Type mismatch: cannot convert from Object to boolean

	at stsjorbsmod.powers.MirroredTechniquePower.getIncomingAttackCount(MirroredTechniquePower.java:62)
	at stsjorbsmod.powers.MirroredTechniquePower.onUseCard(MirroredTechniquePower.java:43)
	at com.megacrit.cardcrawl.actions.utility.UseCardAction.<init>(UseCardAction.java:35)
	at com.megacrit.cardcrawl.characters.AbstractPlayer.useCard(AbstractPlayer.java:1695)
	at com.megacrit.cardcrawl.actions.GameActionManager.getNextAction(GameActionManager.java:358)
	at com.megacrit.cardcrawl.actions.GameActionManager.update(GameActionManager.java:175)
	at com.megacrit.cardcrawl.rooms.AbstractRoom.update(AbstractRoom.java:325)
	at com.megacrit.cardcrawl.dungeons.AbstractDungeon.update(AbstractDungeon.java:2524)
	at com.megacrit.cardcrawl.core.CardCrawlGame.update(CardCrawlGame.java:881)
	at com.megacrit.cardcrawl.core.CardCrawlGame.render(CardCrawlGame.java:429)
	at com.badlogic.gdx.backends.lwjgl.LwjglApplication.mainLoop(LwjglApplication.java:225)
	at com.badlogic.gdx.backends.lwjgl.LwjglApplication$1.run(LwjglApplication.java:126

**Describe the bug**
After using Mirrored Technique the game crashed with the next card I cast

**To Reproduce**
Steps to reproduce the behavior:
1. Enter combat
2. Play Mirrored Technique
3. Play another card
4. Game Crashes

--- a/src/main/java/stsjorbsmod/powers/MirroredTechniquePower.java
+++ b/src/main/java/stsjorbsmod/powers/MirroredTechniquePower.java
@@ -59,7 +59,7 @@ public class MirroredTechniquePower extends CustomJorbsModPower {
                 if (!m.isDeadOrEscaped() && IntentUtils.isAttackIntent(m.intent)) {
 
                     int multiAmt = 0;
-                    if (ReflectionUtils.getPrivateField(m, AbstractMonster.class, "isMultiDmg")) {
+                    if (ReflectionUtils.getPrivateField(m, AbstractMonster.class, "isMultiDmg").equals(true)) {
                         multiAmt = ReflectionUtils.getPrivateField(m, AbstractMonster.class, "intentMultiAmt");
                     }
                     else {

--- a/src/main/java/stsjorbsmod/powers/MirroredTechniquePower.java
+++ b/src/main/java/stsjorbsmod/powers/MirroredTechniquePower.java
@@ -59,6 +59,7 @@ public class MirroredTechniquePower extends CustomJorbsModPower {
                 if (!m.isDeadOrEscaped() && IntentUtils.isAttackIntent(m.intent)) {
 
                     int multiAmt = 0;
+                    //uses .equals(true) to force getPrivateField to boolean, getting around a compilation error with openJDK
                     if (ReflectionUtils.getPrivateField(m, AbstractMonster.class, "isMultiDmg").equals(true)) {
                         multiAmt = ReflectionUtils.getPrivateField(m, AbstractMonster.class, "intentMultiAmt");
                     }

